### ISSUE TITLE
Wrong link conversion

### ISSUE THREAD

**Issue by gurisko** [noyobo/md2jsonml#11]
Hi @noyobo !

First of all, thank you for the module. I started using it and encountered the following bug:

```javascript
const md2jsonml = require('md2jsonml');
const util = require('util');
console.log(util.inspect(md2jsonml(`[test]`), {depth: null}));
```
The `console.log` should print `'[ \'article\', [ \'p\', \'[test]\' ] ]'` instead of assuming that `[test]` is a link. A string should only be converted into a link if it matches `\[([^\]]+)\]\(([^)]*)\)`

**Comment by noyobo** [comment]
Thanks for your feedback

fixed at version 2.0.1

you can see the test file `link.spec.js`

**Comment by gurisko** [comment]
Thank you!
